packages/pg-v5/commands/credentials/destroy.js

```diff
@@ -15,8 +15,9 @@ async function run(context, heroku) {
   }
 
   let db = await fetcher.addon(app, args.database)
+
   if (util.essentialPlan(db)) {
```

Comment:
should this include `essentialNumPlan` too?
```suggestion
  if (util.essentialPlan(db) || util.essentialNumPlan(db)) {
```

Reply:
I was thinking based on this snippet in util.js we will cover both cases:
`exports.essentialPlan = a => {
  return this.essentialNumPlan(a) || this.legacyEssentialPlan(a)
}`

Reply:
An essential plan is either the essentialNumPlan, or the legacyEssentialPlan. Am I understanding that right?